loginas/utils.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from django.conf import settings as django_settings
+from django.contrib.auth.models import User
+from django.contrib.auth import load_backend, login, logout
+from django.contrib import messages
+from django.core.signing import TimestampSigner, SignatureExpired
+from datetime import timedelta
+
+from . import settings
```

Comment:
I'm a bit uneasy about calling this `settings` as well, could you rename it to `la_settings` or import what you need directly from it?